app/Http/Controllers/CompanyController.php

```diff
@@ -57,7 +69,7 @@ public function show(Company $company)
      */
     public function edit(Company $company)
     {
-        //
+        return \view('company.edit', ['company' => $company]);
```

Comment:
replace to `return view('company.edit', ['company' => $company]);` and use import